modules/analysis-common/src/yamlRestTest/resources/rest-api-spec/test/analysis-common/30_tokenizers.yml

```diff
@@ -311,19 +311,6 @@
     - match:  { detail.tokenizer.tokens.1.token: a/b }
     - match:  { detail.tokenizer.tokens.2.token: a/b/c }
 
-    - do:
```

Comment:
We shouldn't remove these test cases but check that the deprecation warning is returned:
```
      allowed_warnings:
        - The [PathHierarchy] tokenizer name is deprecated and will be removed in a future version.  Please change the tokenizer name to [path_hierarchy] instead.

```

Reply:
I think I will also need
```yaml
---
"path_hierarchy":
  - skip:
      features: "allowed_warnings"
```
in the beginning of the test as well. Right?

Reply:
Yeah, just check an existing tests please (indices.clone/10_basic.yml fe)

Reply:
Hmm... not sure if I need the "skip" section. I did not find the answer in [rest-api-spec/test/README.md](https://github.com/opensearch-project/OpenSearch/blob/main/rest-api-spec/src/main/resources/rest-api-spec/test/README.md). I will omit it for now...

Reply:
My understanding is that the `- skip:` condition controls execution of whole test section (so either all `- do:`s or none of it). See https://github.com/opensearch-project/OpenSearch/blob/main/rest-api-spec/src/main/resources/rest-api-spec/test/README.md#skipping-tests

It also reads:
> Once all runners have implemented the feature, it can be declared supported by default, thus the related skip sections can be removed from the tests.

🤔 

If you think that we still need it then I will add it.

Reply:
> Hmm... not sure if I need the "skip" section. I did not find the answer in [rest-api-spec/test/README.md](https://github.com/opensearch-project/OpenSearch/blob/main/rest-api-spec/src/main/resources/rest-api-spec/test/README.md). I will omit it for now...

You will need skip section for version before 2.99.99 because this change will be only in 3.0.0

Reply:
True!